package.json

```diff
@@ -6,13 +6,17 @@
   "author": "Nick Kozlarek <kozlarek.nicholas@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.1.2",
+    "@fortawesome/free-regular-svg-icons": "^6.1.2",
+    "@fortawesome/free-solid-svg-icons": "^6.1.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "axios": "^0.27.2",
     "bluebird": "^3.7.2",
     "chai": "^4.3.6",
     "dotenv": "^16.0.1",
     "eslint": "^8.22.0",
     "express": "^4.18.1",
-    "jest": "^28.1.3",
+    "jest": "^29.0.1",
```

Comment:
In theory, I don't think you are supposed to update the dependencies (Jest was updated from 28.1.3)